src/Microsoft.Data.Sqlite/SqliteDataReader.cs

```diff
@@ -128,11 +125,6 @@ public override bool NextResult()
 #if NET451
         // TODO: Remove when fixed in System.Data.Common
         public override void Close() => Dispose(true);
-
-        public override DataTable GetSchemaTable()
```

Comment:
Ah, is that what the failures are? I thought they updated to net451 contracts...